mitm_proxy/src/mitm_proxy.rs

```diff
@@ -208,9 +208,10 @@ impl MitmProxy {
                 header.col(|_ui| ());
             })
             .body(|mut body| {
+                let mut requests = self.requests.clone();
```

Comment:
For now this way could be a good solution, this because this App isn't performance intensive, I prefer simplicity and a good readability